django_celery_results/settings.py

```diff
@@ -0,0 +1,16 @@
+from django.conf import settings
+
+
+def get_callback_function(settings_name, default=None):
+    """Return the callback function for the given settings name."""
+    
+    callback = getattr(settings, settings_name, None)
+    if callback is None:
+        return default
+
+    if callable(callback):
+        return callback
+
+extend_task_props_callback = get_callback_function(
+    "CELERY_RESULTS_EXTEND_TASK_PROPS_CALLBACK"
+)
```

Comment:
Consider adding a default callable returning an empty dict so this is an opt-in.
Currently this is mandatory if extended_result=True

Reply:
Will be done in the next commit